### DISCUSSION TITLE
0.829 Release Date

### DISCUSSION THREAD

**Discussion by jonboiser** [fabrizioschiavi/pragmatapro#191] [Q&A]
Hi Fabrizio,

I was wondering if you could give us some updates on the next version of the font. Based on the yearly schedule of previous releases, I was really hoping it would have been released in 2020.

**Comment by fabrizioschiavi** [comment]
You've right @jonboiser and I'm so sorry for the delay. I was too busy to work at the update during 2020. Anyway I started to update PragmataPro since december and I hope to relase the new version very soon. Thanks for the patience.
